lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -8243,6 +8243,20 @@ namespace Js
             return true;
         }
 
+        // Since the equivTypes cache is always compacted, if last entry is non-nullptr, it means
+        // the cache is full.
```

Comment:
nit: no need to break this comment to multiple lines since the code following is longer.